src/entities/comment/models/api.ts

```diff
@@ -0,0 +1,52 @@
+import { api } from "../../../shared/api/client";
+import type { Comment, CommentsResponse } from "./types";
+
+interface GetCommentsParams {
+  postId: number;
+}
+export const getComments = async ({ postId }: GetCommentsParams): Promise<CommentsResponse> => {
+  return api<CommentsResponse>(`/comments/post/${postId}`);
+};
```

Comment:
api 에 이미 generic 으로 응답 타입을 정의해주고 있어서 이렇게 해볼 수도 있을 것 같습니다.

```suggestion
export const getComments = async ({ postId }: GetCommentsParams) => {
  return api<CommentsResponse>(`/comments/post/${postId}`);
};
```